spec/screenshots/organizations_portal_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'percy'
+
+RSpec.describe 'Organizations Portal Screenshots', type: :system, js: true do
```

Comment:
Since this is the first thing our customer sees when he logs in, I'd rather not mess it up